### DISCUSSION TITLE
Query & Mutation splitting

### DISCUSSION THREAD

**Discussion by undercoverfemboy** [graphql/graphql-js#3235] [General]
Hello. 
I'm creating a webapp and I have really long RootQuery, with many other queries in it. Is there any way to split code beetwen files?

My code looks like this(it's only a small part of it):
`const RootQuery = new GraphQLObjectType({
    name:'RootQuery',
    fields:{
        getUser:{
            type:UserType,
            resolve(parent,args,request){
             return request.user ? request.user : {errMsg: 'User not logged in'};
            }
        },
       }
})`

**Comment by saihaj** [comment]
There are many ways to split code. [`swapi-graphql`](https://github.com/graphql/swapi-graphql)  is a great example.
